storage/src/main/java/zipkin2/storage/kafka/KafkaSpanConsumer.java

```diff
@@ -64,24 +54,11 @@
       String key = span.traceId();
       byte[] value = SpanBytesEncoder.PROTO3.encode(span);
       calls.add(KafkaProducerCall.create(producer, spansTopicName, key, value));
```

Comment:
I really think it will be better to use list form, especially proto there's no overhead of single-entry list vs not list. opens up later to send partitions of a trace and remains compatible with other tools.